generate.py

```diff
@@ -273,6 +285,9 @@
         if in_table(&DOUBLEWIDE_TABLE, c) {{
             return Self::Two;
         }}
+        if in_table(&ZEROWIDTH_TABLE, c) {{
+            return Self::Combining;
```

Comment:
should be fixed now + the commit message should be less confusing now
(anything else to add?)

Not sure if it's worth to add a "WcWidth::Zero" just for this
workaround, so I used "Combining", which is also width zero :/

Reply:
I think the answer to whether there should be a `Zero` enum variant depends on whether a user of this interface might reasonably want to distinguish between `Zero` and `Combining`.  If the answer is no, then I think `Combining` is fine.
If the answer is yes, then I think the same change should be made to the C and JS implementations and add `{p}_zero` there.

Personally, I think `Combining` feels reasonable, but I wasn't aware of this nuance until this issue was opened, so I'm not an expert :-)